test/issue-test.js

```diff
@@ -27,6 +27,32 @@ describe('Issue', function() {
       });
    });
 
+   describe('#getPreviousPriority()', function() {
+      function issueWithLabels() {
+         var i = new Issue();
+         i.addPreviousLabel('p4', 2);
+         i.addPreviousLabel('p5', 1);
+         i.addPreviousLabel('not-a-priority', 0);
+         i.addLabel('p9', 3);
+         i.addLabel('p2', 2);
+         return i;
+      }
+      it('should choose the most recent previous label', function() {
+         var i = issueWithLabels();
+         assert.equal(4, i.getPreviousPriority());
+         assert.equal(9, i.getPriority());
+      });
+      it('should return null if there is no priority at all', function() {
+         var i = new Issue(1);
```

Comment:
Does it matter that issue #1 doesn't exist on the ifixit/ifixit repo?

Reply:
No, there's no API requests here, this is just a unit test.